akka-projection-slick/src/main/scala/akka/projection/slick/SlickProjection.scala

```diff
@@ -49,16 +58,59 @@ object SlickProjection {
       projectionId: ProjectionId,
       sourceProvider: SourceProvider[Offset, Envelope],
       databaseConfig: DatabaseConfig[P],
-      handler: SlickHandler[Envelope]): ExactlyOnceSlickProjection[Envelope] =
+      handler: SlickHandler[Envelope])(
+      implicit system: ActorSystem[_]): ExactlyOnceSlickProjection[Offset, Envelope] = {
+
+    val offsetStore: SlickOffsetStore[P] =
+      new SlickOffsetStore(databaseConfig.db, databaseConfig.profile, SlickSettings(system))
+
+    // lift to Future handler
+    val liftedHandler =
```

Comment:
not sure if "lifted" is the right name, it's more like a SlickHandlerAdapter?

Reply:
I want to express that we lifted it from DBIO to Future, but adapter you make it more clear that we going to the generic Handler type